packages/cli/src/commands/build.ts

```diff
@@ -316,55 +316,76 @@ export default async function main(client: Client) {
   }
 
   if (!fs.existsSync(join(cwd, OUTPUT_DIR))) {
-    let outputDir = join(OUTPUT_DIR, 'static');
     let distDir = await framework.getFsOutputDir(cwd);
+    const distDirAbsolute = resolve(cwd, distDir);
+    let outputDir = join(OUTPUT_DIR, 'static');
     if (framework.slug === 'nextjs') {
       outputDir = OUTPUT_DIR;
     }
+    const outputDirAbsolute = resolve(cwd, outputDir);
     const copyStamp = stamp();
     await fs.ensureDir(join(cwd, outputDir));
-    const relativeDistDir = relative(cwd, distDir);
+
     client.output.spinner(
       `Copying files from ${param(distDir)} to ${param(outputDir)}`
     );
-    const files = await glob(join(relativeDistDir, '**'), {
-      ignore: [
-        'node_modules/**',
-        '.vercel/**',
-        '.env',
-        '.env.*',
-        '.*ignore',
-        '_middleware.ts',
-        '_middleware.mts',
-        '_middleware.cts',
-        '_middleware.mjs',
-        '_middleware.cjs',
-        '_middleware.js',
-        'api/**',
-        '.git/**',
-        '.next/cache/**',
-      ],
-      nodir: true,
-      dot: true,
-      cwd,
-      absolute: true,
-    });
+
+    // copy files that are going to be mutated later in this script
+    const files = await glob(
+      join(
+        relative(cwd, distDir),
+        '**/{*.nft.json,middleware-manifest.json,required-server-files.json}'
+      ),
```

Comment:
I think this is going to fail on Windows. The docs say to only use forward slashes

https://github.com/isaacs/node-glob#windows

Reply:
If the `cwd` is changed, then the result changes as well, because `middleware-manifest.json` is not in `.next/` as the other files, it's inside `.next/server/`.

Wouldn't it make more sense to instead use something like `path.posix` for this?